AGENTS.md

```
@@ -22,6 +22,10 @@ Project Context documentation can be found in the following locations:
 
 ---
 
+## Output Verification
```

markdownlint-cli2 (0.18.1)

Hard tabs
Column: 1

(MD010, no-hard-tabs)